django/thunderstore/repository/mixins.py

```diff
@@ -1,7 +1,19 @@
+from thunderstore.community.models.community import Community
+
+
 class CommunityMixin:
     @property
     def community_identifier(self):
         return (
             self.kwargs.get("community_identifier", None)
             or self.request.community.identifier
         )
+
+    @property
+    def solved_community_from_identifier(self):
+        return Community.objects.get(identifier=self.community_identifier)
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        context["community_identifier"] = self.community_identifier
```

Comment:
I'm not 100% sure but I feel like some views also want `community` and `community_site`, should they also be added by default in the mixin? Perhaps current views manually defined those, but moving those to the mixin could make more sense.

Reply:
Yes they do and at first I added something like this, but then I backpedaled with it, cause I felt it was unnecessary to include the community query in every get_context_data. Also I wasn't sure which one was better.
1. Faster queries
2. Faster view load times

But now that you mentioned the table join and the fact that views are cached for the most part, I'll revisit this and check what the actual performance cost is the queries.

Reply:
Yeah, alright from what I could figure out, it's a 30% speed increase when switching to using community 👍